e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -1224,6 +1224,128 @@ func (s *IbcEurekaTestSuite) ICS20TimeoutPacketFromEthereumTest(
 	}))
 }
 
+func (s *IbcEurekaTestSuite) TestTimeoutPacketFromCosmos_Groth16() {
+	ctx := context.Background()
+	s.ICS20TimeoutPacketFromCosmosTest(ctx, operator.ProofTypeGroth16, 1)
+}
+
+func (s *IbcEurekaTestSuite) TestTimeoutPacketFromCosmos_Plonk() {
+	ctx := context.Background()
+	s.ICS20TimeoutPacketFromCosmosTest(ctx, operator.ProofTypePlonk, 1)
+}
+
+func (s *IbcEurekaTestSuite) Test_10_TimeoutPacketFromCosmos_Groth16() {
+	ctx := context.Background()
+	s.ICS20TimeoutPacketFromCosmosTest(ctx, operator.ProofTypeGroth16, 10)
+}
+
+func (s *IbcEurekaTestSuite) Test_5_TimeoutPacketFromCosmos_Plonk() {
+	ctx := context.Background()
+	s.ICS20TimeoutPacketFromCosmosTest(ctx, operator.ProofTypePlonk, 5)
+}
+
+func (s *IbcEurekaTestSuite) ICS20TimeoutPacketFromCosmosTest(
```

Comment:
It's interesting that this could also go to `cosmos_to_cosmos` tests. The reason you might wanna do this is because whether the proof type is plonk or groth16 makes no difference here. So I'd say this test should go to cosmos to cosmos because:
1. Cosmos to cosmos tests are faster
2. Cosmos to cosmos tests don't run in POS mode, which these tests wouldn't utilize.
3. Plonk or groth16 makes no difference here.

Reply:
I think that makes sense. For any test that is not Ethereum specific, we should probably just do it cosmos to cosmos to keep it simpler.